src/modules/TaskStore.js

```diff
@@ -0,0 +1,29 @@
+import Task from './Task.js';
+
+export default class TaskStore {
+  constructor() {
+    this.tasksArray = [new Task(1, 'Task 1'), new Task(2, 'Task 2', true)];
+  }
```

Comment:
Great job so far. However, you are required to render the list items in the order of their indexes. 

Consider the code snippet below:

```
 this.tasksArray = [
      new Task(1, "Task 1"),
      new Task(0, "Task 2", true),
      new Task(4, "Task 3", true),
    ].sort((a, b) => a.index - b.index);
```

Reply:
@brytebee your review is subjective. When I render the tasks I sort it. Go look at my render function before I report you to student success.